draft/spec/index.html

```diff
@@ -861,7 +861,32 @@ <h2>Logs Directory</h2>
                 wrong, while others may wish to only store a log entry if an intervention was required.
             </p>
         </blockquote>
+    </section>
 
+    <section id="object-extensions">
+        <h2>Object Extensions</h2>
+        <p>
+            The base directory of an OCFL Object MAY contain a directory named <code>extensions</code> for the purposes
+            of extending the functionality of an OCFL Object. The <code>extensions</code> directory MUST NOT contain
+            any files, and no sub-directories other than extension sub-directories. Extension sub-directories MUST be
+            named according to a registered extension name. The specific structure and function of the extension, as
+            well as a declaration of the registered extension name MUST be defined in one of the following locations:
+            <ul>
+              <li>
+                The <a href="https://ocfl.github.io/extensions/">OCFL Extensions repository</a>
+              </li>
+              <li>
+                The Storage Root as a plain text document
```

Comment:
I can see this leading to confusion that object extensions can be put in the storage root `extensions` directory, not directly in the storage root itself.